synapse/tests/test_lib_httpapi.py

```diff
@@ -404,6 +404,12 @@ async def test_http_auth(self):
                     item = await resp.json()
                     self.eq('ok', item.get('status'))
 
+                await core.setUserLocked(visiiden, True)
+                async with sess.get(f'https://localhost:{port}/api/v1/auth/users', auth=visiauth) as resp:
+                    item = await resp.json()
+                    self.eq('NotAuthenticated', item.get('code'))
+                await core.setUserLocked(visiiden, False)
+
```

Comment:
Since logging is one of the features, if possible, it might be worth having a test to make sure its WAD